std/array.d

```diff
@@ -3173,7 +3174,47 @@ if (isDynamicArray!A)
         }
     }
 
+    /**
+     * Gives a string in the form of `Appender!(A)(data)`.
+     *
+     * Params:
+     *     w = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives).
+     *     fmt = A $(REF FormatSpec, std, format) which controls how the array
+     *     is formatted.
+     * Returns:
+     *     A `string` if `writer` is not set; `void` otherwise.
+     */
+    string toString() const
+    {
+        auto app = appender!string();
```

Comment:
In theory we could also do `const(string) toString()() const { return app.data }` for `appender!string`, but that's a potential optimization for the future.

Reply:
You still need to add `Unqual!(typeof(this)).stringof`

Reply:
Fair point. Though that could be solved by allowing `toString` to return a range, but that's a story for a different PR.